src/ContainerRenameModal.jsx

```diff
@@ -50,7 +50,7 @@ const ContainerRenameModal = ({ container, version, updateContainer }) => {
                     }
                 })
                 .catch(ex => {
-                    setDialogError(cockpit.format(_("Failed to rename container $0"), container.Names[0]));
+                    setDialogError(cockpit.format(_("Failed to rename container $0"), container.Name));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1350-20230712-152457-eee84e1f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)